test-data/unit/check-fastparse.test

```diff
@@ -197,3 +197,12 @@ def f(x):  # E: Type signature has too few arguments
 def g():  # E: Type signature has too many arguments
     # type: (int) -> None
     pass
+
+[case testFastParseMalformedAssert]
+# flags: --fast-parser
+assert 1, 2
+assert (1, 2)  # E: Assertion is always true, perhaps remove parentheses?
+assert (1, 2), 3  # E: Assertion is always true, perhaps remove parentheses?
+# This seems correct, but Python raises a syntax warning for this case so we will too
```

Comment:
I removed it.
To me, `assert (1,2), 3` is unambiguously not a possible mistake. It's clearly two expressions, one of which is a tuple. Now, it's always true, but that's a different issue. So to me, that __shouldn't__ raise the warning, but since the interpreter does, I wanted to replicate that behavior (plus it's simpler).